libs/sdk-ui-ext/styles/internal/scss/attribute_hierarchies.scss

```diff
@@ -102,8 +102,8 @@
         height: 468px;
         margin: 0 -20px;
         padding: 0 20px;
-        border-bottom: 1px solid var(--gd-palette-complementary-3, #dde4eb);
-        border-top: 1px solid var(--gd-palette-complementary-3, #dde4eb);
+        border-bottom: 1px solid #ced8e1;
+        border-top: 1px solid #ced8e1;
```

Comment:
Is it ok to use these colors like that? Recently, there was discussing about this in #frontend channel. Colors should be possible to override via theme.
https://gooddata.slack.com/archives/C01GL9T9R8T/p1700730322597239